Tasks/intersection.js

```diff
@@ -1,25 +1,13 @@
-// Find an intersection of two dictionaries
-
-intersection = function intersection(object_1, object_2) {
-  first_keys = Object.keys(object_1);
-  {
-  }
-  for (attribute_name of first_keys) {
-    if (object_1[attribute_name] === object_2[attribute_name]) {
-      object_2[attribute_name] = object_1[attribute_name];
-      {
-      }
-    } else {
-      delete object_1[attribute_name];
-      {
-      }
-    }
-    {
-    }
-  }
-  {
+'use strict'
+const INTERSECTION =(object_1, object_2)=> {
```

Comment:
Use `camelCase`

Reply:
'use strict';

 const adder = (obj) => {
  let sum = 0;
  const arr_of_values = Object.values(obj);
  for (const element of arr_of_values) {
    if (typeof element === 'number') sum +=   element;
  }
  return sum;
};

require('../Tests/count.js')(adder);

Reply:
'use strict';
const intersection =(object_1, object_2)=> {
  const arr_keys = Object.keys(object_1);
  for (const key of arr_keys) {
    object_1[key]===object_2[key]
    ?object_2[key]=object_1[key]
        :delete object_1[key]
  }
  return object_1;
};

require('../Tests/intersection.js')(intersection);

Reply:
correct variant